learning/inference.py

```diff
@@ -269,6 +275,31 @@ def main():
     inference_data_table = wandb.Table(columns=table_cols, data=inference_data)
     run.log({"Inference Data": inference_data_table})
 
+    video_name = (str(agent.image_directory).split("/"))[-1]
```

Comment:
Use pathlib instead of string manipulation. Check out [PurePath.stem](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.stem)